controllers/openstackcluster_controller.go

```diff
@@ -314,7 +314,8 @@ func reconcileBastion(log logr.Logger, osProviderClient *gophercloud.ProviderCli
 	if err != nil {
 		return err
 	}
-	fp, err := networkingService.GetOrCreateFloatingIP(openStackCluster, openStackCluster.Spec.Bastion.Instance.FloatingIP)
+	clusterName := fmt.Sprintf("%s-%s", cluster.Namespace, cluster.Name)
```

Comment:
Would it be sufficient to just use `cluster` as variable name? Not sure but I guess that the suffix "name" is omitted wherever possible.